com.github.calo001.luna.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "com.github.calo001.luna",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
```

Comment:
I don't see any specific reasons why you're using GNOME Sdk here. You should use FreeDesktop one here. It already has Gtk there. There are not any specific GNOME libs that your app is using

Reply:
Fixed :+1: